importDBdmpFile/1_runDbUpdate.sh

```diff
@@ -173,4 +173,5 @@ cd ..
 # Do not remove following line - another process relies on this exact string
 # being the last line in the log file
 logger "completed ok"
+echo "$(date --iso-8601='seconds') INFO: completed ok"
```

Comment:
It looks a bit weird having this `echo` in the middle of this section bookended by comments about `logger`. Could it go before it?

Reply:
how about [this](https://github.com/GOCDB/gocdb-failover-scripts/pull/21/files/fd67bef3c44412b07ebaef954eab715d1cdd8837#r1339788706)?